app/controllers/places_controller.rb

```diff
@@ -0,0 +1,57 @@
+class PlacesController < ApplicationController
+  def index
+  	@places = Place.all
```

Comment:
inconsistent indentation, we use 2 spaces in Ruby

Reply:
OK